src/common/migrations.js

```diff
@@ -298,6 +298,18 @@ const toV090 = (data) => {
             // eslint-disable-next-line no-param-reassign, prefer-destructuring
             node.data.inputData['2'] = ((node.data.inputData['2'] / 255) * 100.0).toFixed(1);
         }
+
+        // Convert Blur Nodes to Gaussian Blur nodes
+        if (node.data.schemaId === 'chainner:image:blur') {
+            // eslint-disable-next-line no-param-reassign, prefer-destructuring
+            node.data.schemaId = 'chainner:image:gaussian_blur';
+            // eslint-disable-next-line no-param-reassign, prefer-destructuring
+            node.data.inputData['1'] =
+                Math.round((1.16531 * node.data.inputData['1'] - 0.153601) * 10) / 10;
+            // eslint-disable-next-line no-param-reassign, prefer-destructuring
+            node.data.inputData['2'] =
+                Math.round((1.16531 * node.data.inputData['2'] - 0.153601) * 10) / 10;
+        }
```

Comment:
This will cause save to break for people with a dev version of chainner (e.g. you and me). The new migration system I made was added specifically to prevent this.

Move your changes into its own migration function and add that function to the `migrations` array in line 338.

Reply:
I just added it to the existing current migration function with the changes for the adjustment nodes. Did you not integrate that into your new migration system?

Reply:
Yeah, I can confirm that both of these migrations function fine. If I save in dev version, clear, and load the save, the nodes are not changed.

Reply:
>  I can confirm that both of these migrations function fine.

No, they don't. With the new migration, add a Blur node (the one you fixed), save the file, open the file, and see how your new blur node got converted into a gauss node.

Please you [the new migration system](https://github.com/joeyballentine/chaiNNer/pull/331) as intended. It guarantees that this will never be a problem ever again, no matter what the migration is.